File: 717_1bitAnd2bitCharacters/solution.py
class Solution(object):
    def isOneBitCharacter(self, bits):
        i = 0
        while(i < len(bits)):
            if(i == len(bits) - 1):
                return bits[i] == 0
            elif(bits[i] == 1):
                i += 2
            else:
                i += 1
        return False

        """
        :type bits: List[int]
        :rtype: bool
        """
        

File: 717_1bitAnd2bitCharacters/solution.c
bool isOneBitCharacter(int* bits, int bitsSize) {
    int i = 0;
    while(i < bitsSize){
        if(i == bitsSize-1){
            return (bits[i] == 0);
        }
        else if(bits[i] == 1){
            i = i+2;
        }
        else{
            i++;
        }
    }
    return false;
}
